src/code/z_actor.c

```diff
@@ -3755,8 +3789,7 @@ void Actor_AddQuakeWithSpeed(PlayState* play, s16 verticalMag, s16 countdown, s1
     Quake_SetCountdown(quakeIndex, countdown);
 }
 
-// Actor_RequestRumble?
-void func_800BC848(Actor* actor, PlayState* play, s16 y, s16 countdown) {
+void Actor_RequestRumble(Actor* actor, PlayState* play, s16 y, s16 countdown) {
```

Comment:
This conflicts with https://github.com/zeldaret/mm/pull/1129 where this is named `Actor_RequestQuakeAndRumble` (this function will result in both a quake and a rumble).

That PR also names `Actor_RequestQuake` and `Actor_RequestQuakeWithSpeed`

Reply:
Ohh, I had the vague memory of this function already being named.
What do you prefer we should do? Should I unname this function or rename it to `Actor_RequestQuakeAndRumble`?